src/viam/sdk/robot/service.cpp

```diff
@@ -30,6 +30,11 @@ using google::protobuf::RepeatedPtrField;
 using viam::common::v1::ResourceName;
 using viam::robot::v1::Status;
 
+RobotService_::RobotService_(std::shared_ptr<ResourceManager> manager, Server& server)
+    : ResourceServer(std::move(manager)) {
+    server.register_service(this);
+}
```

Comment:
Unlike other resources, `RobotService_` has no `create_resource_server` method, so I opted to `register_service` in the constructor. No one seems to use `RobotService_`, so this is really only relevant for `test_robot.cpp`.